src/collective/taxonomy/configure.zcml

```diff
@@ -9,8 +9,6 @@
     xmlns:zcml="http://namespaces.zope.org/zcml"
     i18n_domain="collective.taxonomy">
 
-  <includeDependencies package="." />
```

Comment:
This should be replaced with the packages you want to include, am I wrong?
They are probably a subset of the requirements:
https://github.com/collective/collective.taxonomy/pull/115/files#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7L48

Reply:
@ale-rt I don't think it needs anything from setup.py. I've tested it on Plone 6 with pip and it seems to work without issues.